gossip/c_block_callbacks.go

```diff
@@ -311,11 +310,25 @@ func consensusCallbackBeginBlockFn(
 
 					_ = evmProcessor.Execute(txs, false)
 
+					startOfBlockWriting := time.Now()
+
 					evmBlock, skippedTxs, allReceipts := evmProcessor.Finalize()
```

Comment:
@rus-alex The `evmProcessor.Finalize` is included into the `blockWriterTimer`, but also into `blockExecutionTimer` - as it includes `statedb.Commit`, I suggest to include it only into `blockWriterTimer` - move the `blockExecutionTimer.Update` bellow before the `startOfBlockWriting`.

Reply:
ok